src/main/java/frc/robot/subsystems/AlgaeIntake.java

```diff
@@ -43,7 +46,7 @@ public void setAlgaeIntakeMotor(double speed) {
   }
 
   public void setAlgaePivotAngle(Angle setpoint) {
-    intakePivotMotor.setPosition(setpoint);
+    intakePivotMotor.setControl(motionRequest.withPosition(setpoint.in(Units.Degrees)));
```

Comment:
We dont need to convert the units anymore :)
```suggestion
    intakePivotMotor.setControl(motionRequest.withPosition(setpoint));
```